AncestralAllele.pm

```diff
@@ -41,18 +41,18 @@ limitations under the License.
  bgzipped file that can be accessed via Bio::DB::HTS::Faidx (installed by VEP's
  INSTALL.pl):
 
- wget https://ftp.ensembl.org/pub/current_fasta/ancestral_alleles/homo_sapiens_ancestor_GRCh38.tar.gz
+ wget ftp://ftp.ensembl.org/pub/current_fasta/ancestral_alleles/homo_sapiens_ancestor_GRCh38.tar.gz
```

Comment:
It added extra change to this line. It should be `https`. I think you can add another commit to fix this.

Reply:
Done